go.mod

```diff
@@ -142,3 +142,8 @@ require (
 	modernc.org/strutil v1.1.3 // indirect
 	modernc.org/token v1.1.0 // indirect
 )
+
+replace (
```

Comment:
NOTE that I'm merging this PR into the worker-versioning feature branch, we should keep these replacements until [this](https://github.com/temporalio/sdk-go/pull/1089) Go SDK change is published and we can merge the API changes.